### ISSUE TITLE
Admin page 404 when pico is in subfolder

### ISSUE THREAD

**Issue by KokoDoko** [gilbitron/Pico-Editor-Plugin#15]
I am testing out Pico in a subfolder of my site :www.mysite.com/pico
The editor is not reachable at either mysite.com/pico/admin or mysite.com/admin
I have named the editor folder "pico_editor" inside the plugins folder.

**Comment by w-vi** [comment]
Strange as I've got it in subfolder as well and login works fine. Have you set the base_url in pico config correctly to www.mysite.com/pico ?

**Comment by KokoDoko** [comment]
Yes, the rest of the site and navigation is working from the sub folder. Only the admin tool isn't working.

**Comment by juanbrujo** [comment]
@KokoDoko first time it didn't worked either, double checked .htaccess and now it works fine.

make sure this plugin files are in plugins/pico_editor/ directory,

**Comment by ekajogja** [comment]
Hi, same issue here, example.com/blog/admin leads to 404. Base url and folder name done.
@juanbrujo can you tell us what we need to change in .htaccess ?

**Comment by nisaea** [comment]
Hello, bumping the thread cause I'm having the same 404 issue. Anyone knows what to change in the .htaccess?

**Comment by juanbrujo** [comment]
Sorry this was many months ago, so I don't have it anymore. I'll try to replicate it.

**Comment by w-vi** [comment]
```
<IfModule mod_rewrite.c>
DirectoryIndex index.php
RewriteEngine on
RewriteBase /~wvi/
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d 
RewriteRule . index.php [L]
</IfModule>

# Prevent file browsing
Options -Indexes
```

This is my htacces and I access the site by localhost/~wvi/ ... everything works fine. Don't know if this helps but the RewriteBase is the most important bit I think.

**Comment by nisaea** [comment]
Ok I had a supermassive facepalm moment there. In my httpd.conf I AllowOverride'd the wrong directory...
Thank you very much for your help and the quick reply! (and sorry for being an idiot)
